File: week2/day3/printer.py

# - Create a function called `printer`
#   which prints the input parameters
#   (can have multiple number of arguments)

a1 = input("Enter 4 numbers to print. Enter first number please: ")
a2 = input("Enter Second number please: ")
a3 = input("Enter third number please: ")
a4 = input("Enter fourth number please: ")

def printer(a1, a2, a3, a4):
    list=[a1, a2, a3, a4]
    print(list)

printer(a1, a2, a3, a4)

File: week8/day3/printer.js
'use strict';
// - Create a function called `printer`
//   which logs to the console the input parameters
//   (can have multiple number of arguments)

function printer() {
    for (var i = 0; i < arguments.length; i++) {
        console.log(arguments[i]);
    }
}
printer('mate', '2');
